jina/peapods/pods/__init__.py

```diff
@@ -593,4 +593,4 @@ def _parse_base_pod_args(self, args):
             parsed_args['peas'] = [args]
 
         # note that peas_args['peas'][0] exist either way and carries the original property
-        return parsed_args
+        return parsed_args
```

Comment:
Please don't remove end of file newlines.

Reply:
Sorry, that was unintentional. I have fixed this in the most recent commit.